Atomic/syscontainers.py

```diff
@@ -306,6 +346,19 @@ def get_system_containers(self):
             ret.append(container)
         return ret
 
+    def delete_image(self, image):
+        repo = self._get_ostree_repo()
```

Comment:
I would prefer that this not return true/false, since it should really throw an exception on failure.  And caller should not call if it is not an ostree image.